src/test/librbd/fsx.cc

```diff
@@ -2941,6 +3174,9 @@ usage(void)
 #if defined(__linux__)
 "	-M: enable rbd-nbd mode (use -t and -h too)\n"
 #endif
+#if defined(_WIN32)
+"	-M: enable rbd-wnbd mode (use -L, -O, -r and -w too)\n"
```

Comment:
About ``-L`` (extended operations), there were a few issues:

* we don't currently support 0 sized disk attachments, IIRC that was the case with ``-L``
* haven't implemented the trim/discard operation as part of the test but it could be disabled using ``-H``
* ``COMPARE_AND_WRITE`` and ``WRITE_SAME`` are currently unsupported by the driver but I see that those operations are optional

I'll double check it and see which flags we can use.

Reply:
I dropped the ``-O`` parameter from the help string. Looks like we still need ``-L`` for now because of the 0 sized attachments.

I'll see if we can drop this constraint from the wnbd driver, then it will be a matter of updating the parameter list. IMO we can do it later as a separate PR.

Reply:
> I'll see if we can drop this constraint from the wnbd driver, then it will be a matter of updating the parameter list. IMO we can do it later as a separate PR.

Sounds good.